src/styles/blockCard.scss

```diff
@@ -0,0 +1,136 @@
+.container {
+  display: grid;
+  max-width: 1200px;
+  grid-template-columns: repeat(1, 200px);
```

Comment:
you can store the value of 200px as variable and reuse it here and in other places.